### ISSUE TITLE
논문 공부

### ISSUE THREAD

**Issue by mingxoxo** [mingxoxo/open-cv#11]
Siamese Neural Networks for One-shot Image Recognition(샴 네트워크)
: https://jayhey.github.io/deep%20learning/2018/02/06/saimese_network/
https://www.edwith.org/deeplearningai4/lecture/34912/
loss function
: https://blog.naver.com/infoefficien/221254432488
Siamese기반 추적 기법 -> 빠른 속도 , 낮은 성능
낮은 성능 -> 첫 프레임의 물체 외형으로 현재 프레임에서의 물체를 찾는 것으로 외형변화를 무시하기 때문에 외형변화가 심할 때 물체 추적을 실패하는 경우가 많음

SiamFC : http://openresearch.ai/t/siamesefc-fully-convolutional-siamese-networks-for-object-tracking/200
AlexNet https://dalpo0814.tistory.com/23
CNN : http://taewan.kim/post/cnn/#fnref:1
Batch normalization: https://sacko.tistory.com/44?category=632408

**Comment by sejongleejeongmin** [comment]
![image](https://user-images.githubusercontent.com/44529556/53289002-790c7f00-37d3-11e9-9413-5166ff1120fc.png)
https://sacko.tistory.com/37

**Comment by sejongleejeongmin** [comment]
![image](https://user-images.githubusercontent.com/44529556/53298978-5c765280-3878-11e9-94b6-f9f20593379c.png)
https://blog.naver.com/varkiry05/221065577703

**Comment by sejongleejeongmin** [comment]
![image](https://user-images.githubusercontent.com/44529556/53317175-f3411e80-390d-11e9-8b00-4d092b1d4b3a.png)
https://en.wikipedia.org/wiki/Loss_functions_for_classification#Logistic_loss

**Comment by sejongleejeongmin** [comment]
![Uploading 논문1.jpg…]()

![2](https://user-images.githubusercontent.com/44529556/53475314-fc1f2500-3ab2-11e9-8e27-c55e4d167a92.jpg)

![3](https://user-images.githubusercontent.com/44529556/53475316-fcb7bb80-3ab2-11e9-98ef-074ddcb67881.jpg)
